README.md

```diff
@@ -79,20 +92,48 @@ Add the following rack middleware to your app:
       end
     end
 
-### Other
+### Rails Applications
```

Comment:
This is based on the ruby example (the ruby one could work with rails apps actually). How about putting it before ruby, so people don't start including ruby and then only notice there is something doable specifically for rails.